sed -i "/SBINDIR/s#sbin#bin#" Make.Rules
sed -i 's/CFLAGS :=/CFLAGS +=/g' Make.Rules
sed -i 's/LDFLAGS :=/LDFLAGS +=/g' Make.Rules
sed -i 's,BUILD_GPERF := ,BUILD_GPERF := no #,' Make.Rules
sed -i '/test:/,/.sh/d' -e '/tests/d' -i Makefile

diff -ruN libcap-2.28-orig/Make.Rules libcap-2.28/Make.Rules
--- libcap-2.28-orig/Make.Rules	2019-12-08 05:10:11.000000000 +0900
+++ libcap-2.28/Make.Rules	2019-12-14 04:17:33.209208501 +0900
@@ -32,7 +32,7 @@
 # Target directories
 
 MANDIR=$(man_prefix)/man
-SBINDIR=$(exec_prefix)/sbin
+SBINDIR=$(exec_prefix)/bin
 INCDIR=$(inc_prefix)/include
 LIBDIR=$(lib_prefix)/$(lib)
 PKGCONFIGDIR=$(prefix)/$(lib)/pkgconfig
@@ -49,9 +49,9 @@
 IPATH += -fPIC -I$(KERNEL_HEADERS) -I$(topdir)/libcap/include
 
 CC := gcc
-CFLAGS := -O2 -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64
+CFLAGS += -O2 -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64
 BUILD_CC := $(CC)
-BUILD_CFLAGS := $(CFLAGS) $(IPATH)
+BUILD_CFLAGS += $(CFLAGS) $(IPATH)
 AR := ar
 RANLIB := ranlib
 DEBUG = -g #-DDEBUG
@@ -60,11 +60,11 @@
         -Wstrict-prototypes -Wmissing-prototypes \
         -Wnested-externs -Winline -Wshadow
 LD=$(CC) -Wl,-x -shared
-LDFLAGS := #-g
+LDFLAGS += #-g
 LIBCAPLIB := -L$(topdir)/libcap -lcap
 LIBPSXLIB := -L$(topdir)/libcap -lpsx -lpthread
 
-BUILD_GPERF := $(shell which gperf >/dev/null 2>/dev/null && echo yes)
+BUILD_GPERF := no #$(shell which gperf >/dev/null 2>/dev/null && echo yes)
 
 SYSTEM_HEADERS = /usr/include
 INCS=$(topdir)/libcap/include/sys/capability.h
@@ -79,8 +79,8 @@
 GOROOT := $(shell go env GOROOT)
 GOCGO := $(shell if [ "$(shell go env CGO_ENABLED)" = 1 ]; then echo yes ; else echo no ; fi)
 GOOSARCH := $(shell go env GOHOSTOS)_$(shell go env GOHOSTARCH)
-CGO_CFLAGS := -I$(topdir)/libcap/include
-CGO_LDFLAGS := -L$(topdir)/libcap
+CGO_CFLAGS += -I$(topdir)/libcap/include
+CGO_LDFLAGS += -L$(topdir)/libcap
 CGO_LDFLAGS_ALLOW := -Wl,-wrap,.+
 CGO_REQUIRED=$(shell $(topdir)/go/cgo-required.sh)
 endif
diff -ruN libcap-2.28-orig/Makefile libcap-2.28/Makefile
--- libcap-2.28-orig/Makefile	2019-12-08 04:45:11.000000000 +0900
+++ libcap-2.28/Makefile	2019-12-14 04:16:33.698435039 +0900
@@ -16,7 +16,6 @@
 ifeq ($(GOLANG),yes)
 	$(MAKE) -C go $@
 endif
-	$(MAKE) -C tests $@
 	$(MAKE) -C progs $@
 	$(MAKE) -C doc $@
 	$(MAKE) -C kdebug $@
@@ -34,9 +33,6 @@
 release: distclean
 	cd .. && ln -s libcap libcap-$(VERSION).$(MINOR) && tar cvf libcap-$(VERSION).$(MINOR).tar --exclude patches libcap-$(VERSION).$(MINOR)/* && rm libcap-$(VERSION).$(MINOR)
 
-test: all
-	cd progs && sudo ./quicktest.sh
-
 morganrelease: distclean
 	@echo "sign the tag twice: older DSA key; and newer RSA kernel.org key"
 	git tag -u D41A6DF2 -s libcap-$(VERSION).$(MINOR) -m "This is libcap-$(VERSION).$(MINOR)"
